lib/transaction.js

```diff
@@ -193,6 +193,44 @@ export default class Transaction {
     return this.sign(this.inputs.map(() => privKey));
   }
 
+  static signHashWeb3(web3, hash) {
+    return web3.eth.getAccounts().then(([address]) => {
```

Comment:
sign with the first account only?

Reply:
For MetaMask it always only one account in the list. No matter how many accounts you have.
But it won't harm if we'll add account index as a param with default value `0`